examples/blackjack.py

```diff
@@ -0,0 +1,182 @@
+
+        If a player's cards total more than 21, the player loses.
+
+        Otherwise, it takes the sum of their cards and determines whether
+        to hit or stand based on their current score.
+        """
+        while True:
+            points = sum_hand(player.hand)
+            if points < 17:
+                print("   Hit.")
+                self.hit(player)
+            elif points == 21:
+                print("   {} wins!".format(player.name))
+                sys.exit(0) # End if someone wins
+            elif points > 21:
+                print("   Bust!")
+                break
+            else:  # Stand if between 17 and 20 (inclusive)
+                print("   Standing at {} points.".format(str(points)))
+                self.scores[player.name] = points
+                break
+
+    def find_winner(self):
+        """
+        Finds the highest score, then finds which player(s) have that score,
+        and reports them as the winner.
+        """
+        winners = []
+        try:
+            win_score = max(self.scores.values())
+            for key in self.scores.keys():
+                if self.scores[key] == win_score:
+                    winners.append(key)
+                else:
+                    None
+            winstring = " & ".join(winners)
+            print("And the winner is...{}!".format(winstring))
+        except ValueError:
+            print("Whoops! Everybody lost!")
+
+    def deal(self):
+        """
+        Deals two cards to each player.
+        """
+        i = 1
+        while i <= 2:
+            for p in self.players:
+                newcard = self.deck.draw()
+                p.hand.append(newcard)
+                print("Dealt {} the {}.".format(p.name, str(newcard)))
+            i += 1
+
+
+    def hit(self, player):
+        """
+        Adds a card to the player's hand and states which card was drawn.
+        """
+        newcard = self.deck.draw()
+        player.hand.append(newcard)
+        print("   Drew the {}.".format(str(newcard)))
+
+
+def sum_hand(hand: list):
+    """
+    Converts ranks of cards into point values for scoring purposes.
+    'K', 'Q', and 'J' are converted to 10.
+    'A' is converted to 1 (for simplicity), but if the first hand is an ace
+    and a 10-valued card, the player wins with a blackjack.
+    """
+    vals = [card.rank for card in hand]
+    for i in range(0, len(vals)):
+        try:
+            vals[i] = int(vals[i])
+        except ValueError:
+            if vals[i] in ['K', 'Q', 'J']:
+                vals[i] = 10
+            elif vals[i] == 'A':
+                vals[i] = 1  # Keep it simple for the sake of example
+    if vals == [1, 10] or vals == [10, 1]:
+        print("   Blackjack!")
+        return(21)
+    else:
+        points = sum(vals)
+        print("   Current score: {}".format(str(points)))
+        return(points)
+
+# Copy-pasted from examples/poker.py
+def generate_deck() -> List[PokerCard]:
```

Comment:
Somebody on reddit posted an interesting and different way of creating a classic deck [here](https://www.reddit.com/r/learnpython/comments/52n1i0/pycarddeck_looking_for_first_time_contributors/d7lm483) = it'd be cool to showcase this instead of repeating what we have in another example. Alternatively use library's `load_standard_deck`